src/test/java/LoginTests.java

```diff
@@ -15,7 +15,7 @@ public static void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://apps.testpro.io/";
+        String url = "testpro.io/";
```

Comment:
**Test failed!** Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/220551669-f1f402c7-2023-47df-bd57-1003cc43ff92.png)

Hint 

`String url = "https://testpro.io/";`



![image](https://user-images.githubusercontent.com/104761482/220552113-aee85c44-e274-47ce-8287-47e87ea1ac7f.png)